samples/bluetooth/unicast_audio_server/src/main.c

```diff
@@ -18,19 +18,39 @@
 
 #define MAX_PAC 1
 
+#define AVAILABLE_SINK_CONTEXT        (BT_AUDIO_CONTEXT_TYPE_UNSPECIFIED | \
+BT_AUDIO_CONTEXT_TYPE_CONVERSATIONAL | BT_AUDIO_CONTEXT_TYPE_MEDIA | \
+BT_AUDIO_CONTEXT_TYPE_GAME | BT_AUDIO_CONTEXT_TYPE_INSTRUCTIONAL)
+#define AVAILABLE_SOURCE_CONTEXT      (BT_AUDIO_CONTEXT_TYPE_UNSPECIFIED | \
+BT_AUDIO_CONTEXT_TYPE_CONVERSATIONAL | BT_AUDIO_CONTEXT_TYPE_MEDIA | \
+BT_AUDIO_CONTEXT_TYPE_GAME)
+
 /* Mandatory support preset by both client and server */
 static struct bt_audio_lc3_preset preset_16_2_1 = BT_AUDIO_LC3_UNICAST_PRESET_16_2_1;
 
 NET_BUF_POOL_FIXED_DEFINE(tx_pool, 1, CONFIG_BT_ISO_TX_MTU, NULL);
 static struct bt_conn *default_conn;
 static struct bt_audio_stream streams[MAX_PAC];
 
+
+static uint8_t unicast_server_addata[] = {
+	BT_UUID_16_ENCODE(BT_UUID_ASCS_VAL), /* ASCS UUID */
+	0x01, /* Target Announcement */
```

Comment:
We should also have a `#define` for the target announcement

Reply:
> We should also have a `#define` for the target announcement

Great, I will add the marco in audio.h